variables.tf

```
@@ -4,26 +4,39 @@
 # You must provide a value for each of these parameters.
 # ------------------------------------------------------------------------------
 
-variable "subnet_id" {
-  type        = string
-  description = "The ID of the AWS subnet to deploy into (e.g. subnet-0123456789abcdef0)"
+variable "users" {
+  type        = map(map(list(string)))
+  description = "A map containing the usernames of each Domain Manager user and a list of roles assigned to that user.  The only currently-defined role is \"provisioner\".  Example: { \"firstname1.lastname1\" = { \"roles\" = [ \"provisioner\" ] } }"
 }
 
 # ------------------------------------------------------------------------------
 # OPTIONAL PARAMETERS
 #
 # These parameters have reasonable defaults.
 # ------------------------------------------------------------------------------
-variable "ami_owner_account_id" {
+
+variable "assume_access_dm_terraform_backend_policy_description" {
+  type        = string
+  description = "The description to associate with the IAM policy that allows assumption of the role that allows access to Domain Manager-related Terraform backend resources."
+  default     = "Allow assumption of the AccessDomainManagerTerraformBackend role in the Terraform account."
```

Given that the policy name could be different is the user chose to override the appropriate variable's default value, the description here should be:
```suggestion
  default     = "The IAM policy that allows assumption of the role that allows access to Domain Manager-related Terraform backend resources."
```